electron/main.js

```diff
@@ -136,16 +162,19 @@ const showMainWindow = () => {
       preload: PRELOAD_JS,
       webviewTag: true,
     },
-    width: config.WINDOW.MAIN.DEFAULT_WIDTH,
-  });
+    width: mainWindowState.width,
+    x: mainWindowState.x,
+    y: mainWindowState.y,
+  };
 
-  const isFullScreen = settings.restore(SETTINGS_TYPE.FULL_SCREEN, false);
-  const windowBounds = settings.restore(SETTINGS_TYPE.WINDOW_BOUNDS, main.getBounds());
-  main.setBounds(windowBounds);
-  if (isFullScreen) {
+  main = new BrowserWindow(options);
+  mainWindowState.manage(main);
+  if (typeof mainWindowState.isMaximized === 'undefined' && isFullScreen === true) {
     main.maximize();
   }
 
+  upgradeSettingsToV1();
```

Comment:
This should only be done if there is an `init.json` file and if that file does not contain a `configVersion` property.